### PR TITLE
Render card data as a JSON file and use it to load cards.

### PR DESCRIPTION
**Pull request by kberg**
This PR is optional based on whether 'experimental ui' is on. Once
people get a chance to try this without any issues and the flag is
removed, the front-end will no longer rely on ICard.

You can see this in action on terraforming-mars-kberg.herokuapp.com.
On that site you don't even need to set a flag, it's on by default.

### CODE REVIEW

File: src/tsconfig.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
     "include": [
         "**/*.ts",
         "genfiles/*.json",
-    ]
+    ],
+    "exclude": ["client/**/*.ts"],
```
Comment (kberg) [inline src/tsconfig.json:10]: Looks as though these .ts files get properly compiled by webpack.

File: package.json [hunk 2]
```diff
@@ -69,13 +69,14 @@
   "scripts": {
     "start": "node build/src/server.js",
     "build": "npm run make:static && npm run build:server && npm run build:client",
-    "build:client": "cross-env NODE_ENV=production webpack",
+    "build:client": "npm run make:cards && cross-env NODE_ENV=production webpack",
```
Comment (kberg) [inline package.json:72]: This PR does change the build process. Now:

1. The server code, which contains code information, has to be built.
2. Using that server code, run it to get information to render the card as JSON.
3. Only then will the front-end compile.

Comment (Argeento) [reply package.json:72]: `cross-env NODE_ENV=production` probably should be the first command in this line

File: src/tsconfig.json (hunk 1 above)
Comment (bafolts) [reply src/tsconfig.json:10]: This will decrease the build time marginally.

**Review by bafolts — APPROVED** [review]

**Pull request merged**